src/rpc/misc.cpp

```diff
@@ -366,6 +366,40 @@ static UniValue setmocktime(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+#ifdef HAVE_BOOST_PROCESS
+static UniValue runcommand(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.empty() || request.params.size() > 2)
+        throw std::runtime_error(
+            RPCHelpMan{"runcommand",
+                "\nRun command and parse JSON from stdout (-regtest only)\n",
+                {
+                    {"command", RPCArg::Type::STR, RPCArg::Optional::NO, "Command must return JSON"},
+                    {"stdin", RPCArg::Type::STR,  /* default */ "", "Pass via stdin"},
+                },
+                RPCResults{},
+                RPCExamples{""},
+            }.ToString()
+        );
+
+    if (!Params().MineBlocksOnDemand())
```

Comment:
Can we check for `-regtest` explicitly instead of implicitly via `fPowNoRetargeting`? Explicit feels better than implicit when it comes to enabling remote execution :)

Reply:
Done